tools/build_config/build.sh

```diff
@@ -85,6 +85,41 @@ function run_package() {
   echo_and_run "cpack -G TGZ"
 }
 
+function build_docs() {
+    # Update release numbers
+    build_id=$(sed -nr '/CPACK_PACKAGE_FILE_NAME/{s/.*"Horace-([^"]+)".*/\1/p};' ./build/CPackConfig.cmake)
+    sed -i -r 's/(release|version) = .*/\1 = "'${build_id}'"/' ./documentation/user_docs/docs/conf.py
+    make -C ./documentation/user_docs/ html
```

Comment:
this `make` call is the actual build so would benefit from some whitespace above it, at the moment it looks like part of the 'update release numbers' operation and is easily overlooked